content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md

```diff
@@ -83,7 +83,7 @@ fieldManager 是与进行这些更改的参与者或实体相关联的名称。
 A selector to restrict the list of returned objects by their fields. Defaults to everything.
 <hr>
 -->
-根据返回对象的字段限制返回对象列表的选择器。默认为返回所有字段。
+根据返回对象的字段限制返回对象列表的选择器。默认为返回所有对象。
```

Comment:
```suggestion
这个选择算符根据对象字段限制返回的对象列表。默认为返回所有对象。
```
The first clause looks like a bit tongue twister, but I have no good ideas yet, only a suggestion for your reference.

Reply:
@seokho-son WDYT of this?

Reply:
Not sure why @NitishKumar06 you mention @seokho-son here.

He is a Korean speaker, not a Chinese speaker.

Reply:
I am not eligible for reviewing this PR. :)

Reply:
@Sea-n My BAD! I got confused.